### ISSUE TITLE
[Feature Request] Custom App icon

### ISSUE THREAD

**Issue by mohammedmohandyeisa** [Jays2Kings/tachiyomiJ2K#1160]
### Why/User Benefit/User Problem
(explain why this feature should be added)
I love this app so much, but the new icon isn't as clean as the app is (all the love for the designer). There are many apps that allow custom app icons.
![Screenshot_20220210-190555](https://user-images.githubusercontent.com/74384750/153460792-4ab65afb-1ba2-4ec6-b095-64c914b7e991.jpg)

### What/Requirements
(explain how this feature would behave)
Basically you open settings > General > App icon
There will be list of icons with the color scheme of each theme in the app. It will be more flexible if there is (import photo from device) option for more options. 

Sorry i submitted this as mistake but its the same request #1159 

Also if the request is denied please i want to submit icon design kindly 

### Thanks in advance

**Comment by mohammedmohandyeisa** [comment]
> 

So i can't submit a request?

**Comment by CarlosEsco** [comment]
You did, and it was closed because its not getting added.  https://github.com/Jays2Kings/tachiyomiJ2K/issues/1069

**Comment by mohammedmohandyeisa** [comment]
> You did, and it was closed because its not getting added. #1069

No i mean this was my suggestion :/
